src/t_stream.c

```diff
@@ -414,7 +414,7 @@ void streamGetEdgeID(stream *s, int first, int skip_tombstones, streamID *edge_i
  * If 'use_id' is not NULL, the ID is not auto-generated by the function,
  * but instead the passed ID is used to add the new entry. In this case
  * adding the entry may fail as specified later in this comment.
- * 
+ *
```

Comment:
please undo these unintended changes and discipline your editor.
we're trying to keep the blame log easy to follow, and avoid unnecessary merge conflicts between branches and forks.

Reply:
Done. Original commit was reverted and a new, no-whitespace-trimmed commit was added instead. I can squash all this mess to a single commit but will have to force push, LMK if this is required.
Faulty vs code extension was identified and configured correctly.

Reply:
no need for squashes and force-push, it's easier to do incremental review without them, and we're gonna squash-merge it anyway at the end.